extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js

```diff
@@ -95,7 +95,8 @@ export const Presets = {
               const style = computedStyle(fxElement.getAmpDoc().win,
                   fxElement.getElement());
               setStyles(fxElement.getElement(), {
-                'top': `calc(${style.top} + ${fxElement.getFlyInDistance()}vh)`,
+                'top': `calc(${style.top == 'auto' ? '0px' : style.top} +
```

Comment:
While you can put any expression in a template literal, it is probably better to keep things simple and just print variables (or at least avoid too much logic).

Perhaps something like:

```javascript
const topAsLength = style.top == 'auto' ? '0px' : style.top;
setStyles(fxElement.getElement(), {
  'top': `calc(${topAsLength} + ${fxElement.getFlyInDistance()}vh`,
})
```

Reply:
Done in latest patch